switchbot/devices/curtain.py

```diff
@@ -210,3 +247,11 @@ def is_calibrated(self) -> Any:
         """Return True curtain is calibrated."""
         # To get actual light level call update() first.
         return self._get_adv_value("calibration")
+
+    def is_opening(self) -> Any:
+        """Return True if the curtain is opening."""
+        return self._is_opening
+    
+    def is_closing(self) -> Any:
+        """Return True if the curtain is closing."""
+        return self._is_closing
```

Comment:
```suggestion
    @property
    def is_opening(self) -> Any:
        """Return True if the curtain is opening."""
        return self._is_opening

    @property
    def is_closing(self) -> Any:
        """Return True if the curtain is closing."""
        return self._is_closing
```

Reply:
I'm fine with this change but I was trying to follow the existing pattern from is_reversed and is_calibrated. So this would now be inconsistent with the rest of the pattern.

Reply:
OK